### PR TITLE
Add ability to add user-fixes.py to $HOME

### PR DESCRIPTION
**Pull request by yuvipanda**

### CODE REVIEW

File: singleuser/user-fixes.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+custom_path = os.path.expanduser('~/user-fixes.py')
+if os.path.exists(custom_path):
+    with open(custom_path, 'r') as f:
+        exec(compile(f.read(), custom_path, 'exec'), globals())
+
+    del f
+
+# Clean up temp variables, since pwb issues a warning otherwise
+# to help people catch misspelt config
+del custom_path
```
Comment (jayvdb) [inline singleuser/user-fixes.py]: del os also?

Comment (yuvipanda) [inline singleuser/user-fixes.py]: That hasn't actually thrown up errors so far.

File: singleuser/user-fixes.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import os
+
+custom_path = os.path.expanduser('~/user-fixes.py')
+if os.path.exists(custom_path):
+    with open(custom_path, 'r') as f:
```
Comment (xZise) [inline singleuser/user-fixes.py]: Use `rb` and `compile` will understand any encoding the file uses (if it's complying with the Python standard).

Comment (yuvipanda) [inline singleuser/user-fixes.py]: Isn't 'b' a noop when on Linux?

Comment (yuvipanda) [inline singleuser/user-fixes.py]: The mode string can also include the letter 'b' either as a last
       character or as a character between the characters in any of the two-
       character strings described above.  This is strictly for
       compatibility with C89 and has no effect; the 'b' is ignored on all
       POSIX conforming systems, including Linux. 

from http://man7.org/linux/man-pages/man3/fopen.3.html.

**Pull request merged**